awx/main/utils/common.py

```diff
@@ -751,15 +751,16 @@ def convert_cpu_str_to_decimal_cpu(cpu_str):
     return max(0.1, round(cpu, 1))
 
 
-def get_corrected_cpu(cpu_count):  # formerlly get_cpu_capacity
+def get_corrected_cpu(cpu_count, node_type=None):  # formerlly get_cpu_capacity
     """Some environments will do a correction to the reported CPU number
     because the given OpenShift value is a lie
     """
     from django.conf import settings
 
     settings_abscpu = getattr(settings, 'SYSTEM_TASK_ABS_CPU', None)
     env_abscpu = os.getenv('SYSTEM_TASK_ABS_CPU', None)
-
+    if node_type == 'execution':
```

Comment:
I'd recommend checking against the variants of `Instance.Types` going forward instead of strings.

e.g. `if node_type == Instance.Types.EXECUTION:`

This will give an immediate error if you accidentally try to check against a type that doesn't exist, where with the string check you could have a typo and the block just never executes.